### PR TITLE
Issue#2 email handling

### LINKED ISSUE #1: Search for On Boarding Email
* Connect to your Outlook mail account
* Search through all unread/read emails in inbox folder for an email with a specific subject "Weekly On Boarding Candidate List"
* Check if email is present or not
* If emails not present end the process
* If email's present open it
* Check if open email has an excel attachment
* If attachment present download the attachment
* If not present get the senders email address & throw Business Exception #4 

Don't forget to use annotations on major sequences plus renaming your activities, variables & objectives following best practices

### LINKED ISSUE #4: Implement Error Handling for Missing Excel Attachment
If excel attachment isn't present the following happen #1   :-

* Handle System Exception & send email in case an error occurs during runtime
* Handle Business Exception & send an email to the Sender email address informing them that the email was received but doesn't have the excel attachment
* Move the email to a different folder #6  and end the process

Don't forget to use annotations on major sequences plus renaming your activities, variables & objectives following best practices

### PR DESCRIPTION
**Pull request by mbunix**
Tried to break the implementation into new Prs

### CODE REVIEW

File: Emails/GetEmailDetails.xaml
Comment (FreddieMwau) [inline Emails/GetEmailDetails.xaml:1]: Make the following changes 
* stop using hard coded values on Send Outlook Mail Message (Reply To) > BusinessRuleException
image
* Create variables to hold the arguments values in TakeScreenShot.xaml, also don't hard code values on the Send Outlook Mail Message (Reply To) this is in System Exception
image

**Review by FreddieMwau — CHANGES_REQUESTED** [review]
Good work, avoid hard coding values & handle the arguments on the System exception and we're good

**Comment by mbunix** [comment]
Made the changes , watch the orchestrator asset folder

**Review by FreddieMwau — APPROVED** [review]

**Review by FreddieMwau — CHANGES_REQUESTED** [review]
Revert back to how you'd set up the exception emails, they need to be in the Constants Tab & not the Assets Tab

File: Emails/GetEmailDetails.xaml
Comment (FreddieMwau) [inline Emails/GetEmailDetails.xaml:1]: Add the rethrow activity in the Business Exception and this is good

File: Data/Config.xlsx
Comment (FreddieMwau) [inline Data/Config.xlsx:1]: Have the email body and subject save in this excel and not assets 
image

File: Emails/GetEmailDetails.xaml
Comment (FreddieMwau) [reply Emails/GetEmailDetails.xaml:1]: Also change the destination where the attachment will be saved to the shared folder as you suggested

**Review by FreddieMwau — CHANGES_REQUESTED** [review]
Make the changes in the commented xamls

**Review by FreddieMwau — APPROVED** [review]

**Pull request merged**